identity.go

```diff
@@ -1,19 +1,12 @@
 package dbft
 
 import (
-	"encoding"
 	"fmt"
 )
 
 type (
 	// PublicKey is a generic public key interface used by dbft.
-	PublicKey interface {
-		encoding.BinaryMarshaler
-		encoding.BinaryUnmarshaler
-
-		// Verify verifies if sig is indeed msg's signature.
-		Verify(msg, sig []byte) error
-	}
+	PublicKey any
```

Comment:
Looks somewhat weird. But likely it'd be even more weird with this alias removed (you can have `any` everywhere, but...)

Reply:
Agree, but at the same time the library itself doesn't need anything from this interface. And I'm against of removing it, since it's an additional hint to the user if he can make a type cast.

Reply:
Agree to merge?